packages/application-components/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@commercetools-frontend/application-components",
+  "version": "7.0.0",
+  "description": "Generic components for building Merchant Center applications",
+  "bugs": "https://github.com/commercetools/merchant-center-application-kit/issues",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/commercetools/merchant-center-application-kit.git"
+  },
+  "keywords": [
+    "javascript",
+    "frontend",
+    "react",
+    "toolkit"
+  ],
+  "license": "MIT",
+  "private": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "dist/application-components.cjs.js",
+  "module": "dist/application-components.es.js",
+  "files": [
+    "dist",
+    "materials",
+    "package.json",
+    "LICENSE",
+    "README.md"
+  ],
+  "scripts": {
+    "prebuild": "rimraf dist/**",
+    "build": "npm run build:es && npm run build:cjs",
+    "build:cjs": "cross-env NODE_ENV=production rollup -c ../../rollup.config.js  -f cjs ./index.js -o dist/application-components.cjs.js",
+    "build:es": "cross-env NODE_ENV=production rollup -c ../../rollup.config.js  -f es ./index.js -o dist/application-components.es.js",
+    "build:es:watch": "npm run build:es -- -w"
+  },
+  "dependencies": {
+    "prop-types": "15.6.2"
+  },
+  "devDependencies": {
+    "@commercetools-frontend/application-shell": "7.0.0",
```

Comment:
I'm not sure I like this, it introduces a circular dependency. I don't think we need the test utils from app-shell to be honest, we can use RTL directly because there is not much to setup.

I would add a `test-utils` file in this package with only the setup needed for the components (e.g. react-intl etc).